README.md

```diff
@@ -22,19 +24,27 @@ Autometrics for Python provides a decorator that can create [Prometheus](https:/
 
 ## Using autometrics-py
 
-- Requirement: a running [prometheus instance](https://prometheus.io/download/) 
-- include a .env file with your prometheus endpoint ```PROMETHEUS_URL = your endpoint```, if not defined the default endpoint will be ```http://localhost:9090/```
-- ```pip install autometrics```
+- Requirement: a running [prometheus instance](https://prometheus.io/download/)
+- Configure prometheus to scrape your application ([check our instructions if you need help](https://github.com/autometrics-dev#5-configuring-prometheus))
+- Include a .env file with your prometheus endpoint `PROMETHEUS_URL=your endpoint`. If this is not defined, the default endpoint will be `http://localhost:9090/`
+- `pip install autometrics`
 - Import the library in your code and use the decorator for any function:
-```
-from autometrics import autometrics
+
+```py
+from autometrics.autometrics import autometrics
```

Comment:
seems like we need to use `from autometrics.autometrics` - wondering if there's a way to make the import cleaner

Reply:
i'm thinking we might actually do something like:
```
import autometrics

@autometrics
...
```
couldn't we? half as joke but i think it's doable

Reply:
never mind, i forgor how python package resolution works. here's a pr that enables usage like `from autometrics import autometrics`